amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/InterchangeEndpoints.java

```diff
@@ -300,6 +301,17 @@ public PreviewActivityFunding getPreviewFundingInformation(
             @QueryParam(ActivityEPConstants.PREVIEW_CURRENCY_ID) Long currencyId) {
         return PreviewActivityService.getInstance().getPreviewActivityFunding(projectId, currencyId);
     }
+    
+    @GET
+    @Path("/{project-id}/preview/workspaces")
+    @Produces(MediaType.APPLICATION_JSON + ";charset=utf-8")
+    @ApiMethod(id = "getPreviewWorkspaces", ui = false)
+    @ApiOperation(value = "Retrieve workspaces where the activity is visible.")
```

Comment:
Since this is only for managers / approvers of the ws, lets add the restriction so its only accessible by logged in users in a worskpace